docs/pages/2020_BIGMAP/sections/browse.rst

```diff
@@ -0,0 +1,66 @@
+.. _BIGMAP_2020_browse:
+
+Browsing your database
+======================
+
+This final section of the tutorial is optional, and has the purpose of introducing a couple of ways in which you can browse an AiiDA database.
```

Comment:
Is it optional, though? (maybe it has to be 😅)

Reply:
Yeah, maybe it is better to just take it out from the text and just say it if we see there is not enough time.